src/main/java/vn/zalopay/benchmark/util/ExceptionUtils.java

```diff
@@ -0,0 +1,131 @@
+package vn.zalopay.benchmark.util;
+
+import com.alibaba.fastjson.JSONObject;
+
+/**
+ * <b>异常打印工具类</b>
+ * <p>reference: https://github.com/yl-yue/yue-library/blob/j11.2.6.0/yue-library-base/src/main/java/ai/yue/library/base/util/ExceptionUtils.java</p>
+ *
+ * @author	ylyue
+ * @since	2018年9月9日
+ */
+public class ExceptionUtils {
+
+	@SuppressWarnings("unchecked")
+	private synchronized static <T> T getPrintException(Throwable e, Integer line, ExceptionConvertEnum exceptionConvertEnum) {
+		T msg = null;
+		if (exceptionConvertEnum == ExceptionConvertEnum.JSONObject) {
+			msg = (T) new JSONObject(true);
+			if (e == null) {
```

Comment:
Can we reduce the complexity of this code block

Reply:
Hello, this is a copied utility class (copied from other open source projects)

Instead of referencing a JAR package, I simply copied the source code for brevity.